test/wdio/event-custom-type/cmp.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('event-custom', function () {
+  beforeEach(() => {
+    render({
+      template: () => <event-custom-type></event-custom-type>,
+    });
+  });
+
+  it('should dispatch an event on load', async () => {
+    await expect($('#counter')).toHaveText('1');
+  });
+
+  it('should emit a complex type', async () => {
+    // wait to ensure the event it emitted in the component
+    await browser.pause(100);
```

Comment:
This should not be needed as WebdriverIO will retry the assertion in case it fails
```suggestion
```

Reply:
makes sense, thanks!